pyproject.toml

```diff
@@ -76,7 +76,7 @@ wheel = { packages = ["a11y_pygments"] }
 
 [tool.hatch.envs.dev.scripts]
 render_html = "python test/render_html.py"
-create_css = "python test/run_css.py && python test/run_css.py --save-dir docs"
+create_css = "python test/run_css.py --save-dir docs/_build"
```

Comment:
`_build/` is already ignored by the .gitignore file, which is why I decided to use it

Reply:
and it also aligns with other patters we use in the ecosystem